src/archivist/fetcher/fullDomFetcher.js

```diff
@@ -25,7 +24,6 @@ export default async function fetch(url, cssSelectors) {
   try {
     page = await browser.newPage();
 
-    await page.setUserAgent(new UserAgent().toString());
```

Comment:
Could we please use `OpenTermsArchive/${package.version}`? 🙂
I'd like our crawlers to start appearing in logs.

Reply:
Suggestion made in #815.